alerts/openshift-container-storage-operator/StorageAutoScalingCapacityReached.md

```diff
@@ -0,0 +1,34 @@
+# StorageAutoScalingCapacityReached
+
+## Meaning
+
+The Storage Capacity Limit is the maximum storage that can be provisioned for
+the system. `StorageAutoScalingCapacityReached` alert is fired when the complete
+storage space has been utilized and hence additional storage cannot be allocated
+for the device class in the particular storage cluster.
+
+## Impact
+
+The storage cannot be auto-scaled at the moment.
+
+## Diagnosis
+
+Retrieve status of `storageCapacityLimitReached` from the StorageAutoScaler CR.
+The `resource_name` and `resource_namespace` would be specified in the alert itself.
+
+```bash
+oc get storageautoscaler <resource_name> -n <resource_namespace> -o json | jq -r '.status.storageCapacityLimitReached'
+```
+
+If status of `storageCapacityLimitReached` is `true` then maximum capacity of
+the storage has been utilized.
+
+## Mitigation
```

Comment:
Another possible mitigation should be to disable the auto scale, not just to increase the limit. The user needs to decide what they prefer. There are ways to scale out (add more OSDs) without auto-scale enabled.